i18n/i18n_test.go

```diff
@@ -0,0 +1,296 @@
+package i18n_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+	"github.com/mattermost/mattermost-server/v5/plugin/plugintest"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/text/language"
+
+	"github.com/mattermost/mattermost-plugin-api/i18n"
+)
+
+func ExampleInitBundle() {
+	type Plugin struct {
+		plugin.MattermostPlugin
+
+		b *i18n.Bundle
+	}
+
+	p := Plugin{}
+	b, err := i18n.InitBundle(p.API, filepath.Join("assets", "i18n"))
```

Comment:
Just wondering whether we should promote using the plugin api client defined in this repo. Maybe it makes things quite complicated in terms of interfacing the API needed, but it may make sense.

Reply:
1/5 I feel like the api client isn't ready to be used yet. There a still a couple of issues to resolve first. But maybe this topic only needs a final push.